### PR TITLE
Fix dependency properties for Azure Monitor Trace Exporter

### PR DESCRIPTION
**Pull request by lzchen**
Addresses [#14]

data.datais url.
data.target is host:port
data.name is METHOD/path

### CODE REVIEW

**Review by lmolkova — APPROVED** [review]
LGTM

File: azure_monitor/src/azure_monitor/trace.py [hunk 1]
```diff
@@ -134,8 +134,16 @@ def span_to_envelope(self, span):  # noqa pylint: disable=too-many-branches
                 data.type = "HTTP"  # TODO
```
Comment (lmolkova) [inline azure_monitor/src/azure_monitor/trace.py]: oh, would you mind fixing this one too in this PR? Or should I create another issue similar to https://github.com/census-instrumentation/opencensus-python/issues/825?

**Review by lmolkova — COMMENTED** [review]
left a comment about dependency type HTTP for all client spans

File: azure_monitor/src/azure_monitor/trace.py (hunk 1 above)
Comment (lzchen) [reply azure_monitor/src/azure_monitor/trace.py]: I will fix this here! :)

**Pull request merged**